src/card.css

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: var(--color-accent-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", "Times New Roman", serif;
+  font-style: normal;
+
+  --color-main-accent: #060b35;
+  --color-accent-blue: #00acdc;
+  --color-accent-white: #fff;
+  --color-accent-grey: #616070;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__image {
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.name {
+  text-align: start;
+  width: 166px;
+  height: 36px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-accent);
+  order: 0;
+}
+
+.card__name {
+  margin-top: 40px;
+  margin-left: 16px;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-accent-grey);
+  line-height: 14px;
+  order: 1;
+  align-self: flex-start;
+}
+
+.card__code {
+  margin-top: 4px;
+  margin-left: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px 16px 16px;
+  order: 3;
+}
+
+.price__text {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-accent-grey);
+  line-height: 18px;
+}
+
+.price__number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-main-accent);
+}
+
+.card__link {
+  margin-bottom: 16px;
+  margin-left: 16px;
+}
+
+.link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--color-accent-white);
+  background-color: var(--color-accent-blue);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  order: 4;
+}
+
+.link:hover {
+  border: 1px solid var(--color-accent-blue);
```

Comment:
why do we need to change border color on hover? just add this style to your `.link`